src/foam/comics/v2/DAOBrowseControllerView.js

```diff
@@ -123,12 +131,15 @@ foam.CLASS({
         if ( this.config.click && typeof this.config.click === 'function' )
           return this.config.click;
         return function(obj, id) {
-          if ( ! this.stack ) return;
+          if ( ! this.stack ) {
+            console.warn('Missing stack, can not push view');
+            return;
+          }
           this.stack.push(foam.u2.stack.StackBlock.create({
           view: {
             class: 'foam.comics.v2.DAOSummaryView',
             data: obj,
-            config: this.__context__.config,
+            config: this.config || this.__subContext__.config,
```

Comment:
Why isn't this just: this.config?

Reply:
This function is not called in the current view and is exported, old tableViews need to for rowClick to work if i recall correctly

Reply:
I mean, won't this.config and this.__subContext__.config always be the same value, so that this could just be simplified to this.config?